x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/auto_follow_pattern_list.js

```diff
@@ -40,25 +60,20 @@ export const AutoFollowPatternList = injectI18n(
       clearInterval(this.interval);
     }
 
-    componentDidUpdate() {
-      const {
-        openDetailPanel,
-        closeDetailPanel,
-        isDetailPanelOpen,
-        history: {
-          location: {
-            search,
-          },
-        },
-      } = this.props;
-
-      const { pattern: patternName } = extractQueryParams(search);
-
-      // Show deeplinked auto follow pattern whenever patterns get loaded or the URL changes.
-      if (patternName != null) {
-        openDetailPanel(patternName);
-      } else if (isDetailPanelOpen) {
-        closeDetailPanel();
+    componentDidUpdate(_, prevState) {
```

Comment:
Instead of `_` can we use `prevProps`, which is more conventional? It seems unusual and potentially confusing to use `_`.